python/ray/actor.py

```diff
@@ -504,6 +506,16 @@ def _remote(self,
         else:
             detached = False
 
+        if placement_group is None:
```

Comment:
Can we move this block to `placement_group.py` to avoid code duplication? (ex, `ray.experimental.check_placement_group_index(placement_group)`).

Reply:
fixed